### DISCUSSION TITLE
Welcome to Bruno Discussions!

### DISCUSSION THREAD

**Discussion by helloanoop** [usebruno/bruno#268] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pkfox** [comment]
Hi my name is Pete Kane I've been a professional programmer for ~ 40  years ( retired now but still at it ) just discovered Bruno after years with Postman ( which was very good but seems to have lost its way )  the final death knoll for PM was one day all my collections disappeared   - Bruno like anything else has a learning curve but I'm getting there with help from Mr Google et al. What convinced me to stay with Bruno was reading this article [Ryan Reynolds](https://blog.usebruno.com/js-file) I think Bruno caters for a generation of new users and disillusioned PM users like myself  - Nice work guys
